src/storage/copier/npy_reader.cpp

```diff
@@ -0,0 +1,261 @@
+#include "storage/copier/npy_reader.h"
```

Comment:
Is this code from a third-party library? 
If yes, should we move this file under third-party?

Reply:
Moved to third party